tmt/utils.py

```diff
@@ -7270,3 +7271,80 @@ def render_rst(text: str, logger: Logger) -> str:
     document.walkabout(visitor)
 
     return visitor.rendered
+
+
+def jira_link(
+        nodes: list[Union['tmt.base.Test', 'tmt.base.Plan', 'tmt.base.Story']],
+        links: 'tmt.base.Links',
+        separate: Optional[bool] = False) -> None:
+    """ Link the object to Jira issue and create the URL to tmt web service """
+
+    def create_url(tmt_object: 'tmt.base.Core') -> list[str]:
+        url: list[str] = []
+        # Get the fmf id of the object
+        if isinstance(tmt_object, tmt.base.Test):
+            fmfid = tmt_object.fmf_id
```

Comment:
First, I believe you don't have to set `fmf_id` in every branch - all `Core` instances should have it, and indeed, all three lines are exactly the same, `fmfid = tmt_object.fmf_id`.

Second, you may use the class name to generate `tmt_type`, `tmt_object.__class__.__name__.lower()` should give you the same outcome.

Together, you should be able to replace this `if` with two lines handling all possible classes :)